package.json

```diff
@@ -22,11 +22,11 @@
 	},
 	"license": "GPL-3.0+",
 	"scripts": {
-		"deploy": "composer install --no-dev && npm run build --loglevel error && sh ./bin/github-deploy.sh",
+		"deploy": "cross-env WOOCOMMERCE_BLOCKS_PHASE=stable composer install --no-dev && npm run build --loglevel error && sh ./bin/github-deploy.sh",
 		"release": "sh ./bin/wordpress-deploy.sh",
 		"build": "cross-env BABEL_ENV=default NODE_ENV=production webpack",
 		"build:ci": "npm run build && npm run size-check && npm run lint:js && npm run lint:css",
-		"start": "cross-env BABEL_ENV=default webpack --watch --info-verbosity none",
+		"start": "cross-env WOOCOMMERCE_BLOCKS_PHASE=experimental BABEL_ENV=default webpack --watch --info-verbosity none",
```

Comment:
Is this needed if the default is `experimental` (`npm build` or `npm build:ci` executed on their own will be using `experimental` too)?

Reply:
now that we have default values, we don’t need to pass this yeah.